platform/darwin/test/MLNStyleTests.mm

```diff
@@ -233,7 +233,7 @@ - (void)testAddingLayersWithDuplicateIdentifiers {
     XCTAssertThrowsSpecificNamed([self.style insertLayer:[[MLNFillStyleLayer alloc] initWithIdentifier:@"my-layer" source:source] aboveLayer:initial], NSException, @"MLNRedundantLayerIdentifierException");
     XCTAssertThrowsSpecificNamed([self.style insertLayer:[[MLNFillStyleLayer alloc] initWithIdentifier:@"my-layer" source:source] atIndex:0], NSException, @"MLNRedundantLayerIdentifierException");
 #if !MLN_RENDER_BACKEND_METAL
```

Comment:
Can this test be enabled again?

Reply:
```
Test Case '-[MLNStyleTests testAddingLayersWithDuplicateIdentifiers]' started.
Test Case '-[MLNStyleTests testAddingLayersWithDuplicateIdentifiers]' passed (0.007 seconds).
```
it looks like it can be removed.